hudi-common/src/test/java/org/apache/hudi/common/model/TestHoodieRecordDelegate.java

```diff
@@ -70,4 +78,24 @@ public void testKryoSerializeDeserialize() {
     assertEquals(new HoodieRecordLocation("001", "file01"), hoodieRecordDelegate.getCurrentLocation().get());
     assertEquals(new HoodieRecordLocation("001", "file-01"), hoodieRecordDelegate.getNewLocation().get());
   }
+
+  public Kryo getKryoInstance() {
+    final Kryo kryo = new Kryo();
+    // This instance of Kryo should not require prior registration of classes
+    kryo.setRegistrationRequired(false);
+    kryo.setInstantiatorStrategy(new Kryo.DefaultInstantiatorStrategy(new StdInstantiatorStrategy()));
+    // Handle cases where we may have an odd classloader setup like with libjars
+    // for hadoop
+    kryo.setClassLoader(Thread.currentThread().getContextClassLoader());
+
+    // Register Hudi's classes
+    new HoodieCommonKryoRegistrar().registerClasses(kryo);
+
+    // Register serializers
+    kryo.register(Utf8.class, new SerializationUtils.AvroUtf8Serializer());
+    kryo.register(GenericData.Fixed.class, new GenericAvroSerializer<>());
```

Comment:
Do we need a registration for avro classes?

Reply:
No, the member variable types of `HoodieRecordDelegate` don't contain avro.that should't be registered. 
Has been updated.